example/src/store/validate/dynamicDefaults.ts

```diff
@@ -0,0 +1,149 @@
+import dayjs, { UnitType } from 'dayjs';
+import customParsingPlugin from 'dayjs/plugin/customParseFormat';
+import durationPlugin from 'dayjs/plugin/duration';
+import timezonePlugin from 'dayjs/plugin/timezone'; // dependent on utc plugin
+import utcPlugin from 'dayjs/plugin/utc';
+
+// required for the custom save formats in the date, time and date-time pickers
+dayjs.extend(customParsingPlugin);
+dayjs.extend(utcPlugin);
+dayjs.extend(timezonePlugin);
+dayjs.extend(durationPlugin);
```

Comment:
Would other applications using this renderer set also need to configure this? If yes, it should be documented in the README